.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 <!--- Describe the changes here. --->
 
-**Changelog-friendly one-liner**: <!--- One-liner description here --->
+**Changelog-friendly one-liner** <!-- One-liner description here -->
```

Comment:
Thanks for fixing triple dash!

```suggestion
**Changelog-friendly one-liner**: <!-- One-liner description here -->
```

I think the colon shouldn't be removed because it splits the label and text. The template is: 

**Changelog-friendly one-liner**: My awesome feature

See an example #913.

Reply:
Thanks for the comment. Colon reverted.